### DISCUSSION TITLE
geoIp2 for nginx opensource

### DISCUSSION THREAD

**Discussion by tibz7** [nginxinc/ansible-role-nginx#638] [Q&A]
Is there  a way to use this role and install geoip2 for the opensource version? 
Something like  using the source and passing compilation flags? 
Otherwise the only option i thought of was to not use this role and write my own tasks to compile it with the needed module.

**Comment by alessfg** [comment]
We do not build geoip2 for NGINX Open Source, but I don't see why we couldn't tweak the source install option to add the option to link/add dynamic modules during build time (which you'd then use at your own risk). I'll move this to an issue 😄.

**Reply by tibz7** [reply]
that would actually be awsome!
